frontend/src/types.ts

```diff
@@ -682,27 +682,28 @@ export interface TrendResultWithAggregate extends TrendResult {
 }
 
 export interface FunnelStep {
+    // The type returned from the API.
     action_id: string
-    average_conversion_time: number
+    average_conversion_time: number | null
     count: number
     name: string
     order: number
     people: string[]
     type: EntityType
     labels?: string[]
+    breakdown?: string
+    breakdown_value?: string
 }
 
-export interface FunnelResult {
-    is_cached: boolean
-    last_refresh: string | null
-    result: FunnelStep[]
-    type: 'Funnel'
+export interface MappedFunnelStep extends Omit<FunnelStep, 'breakdown'> {
+    // A more useful type which reflects the hierarchy of a breakdown.
```

Comment:
This comment is not very useful in that it doesn't answer the question "why?". Why do we need this MappedFunnelStep? Why is it "more useful"? 

Could there be a better name that describes the difference between FunnelStep and MappedFunnelStep? The prefix "Mapped" on its own doesn't really say anything. How is the data different? I'd even say something very verbose like `FunnelStepWithAggregatedBrekdown` would fit better.

Ideally I would avoid this custom type altogether.